fe1-web/docs/README.md

```diff
@@ -344,6 +346,247 @@ which ensures that all constraints defined within the JSON Schema are respected.
 Furthermore, each object has specific constraints defined within its constructor. For an example, see
 [CreateRollCall.ts](https://github.com/dedis/popstellar/blob/master/fe1-web/src/features/rollCall/network/messages/CreateRollCall.ts)
 
+## Dependencies between features
+
+### Idea
+
+One may think of the features as being independent packages that are loaded by
+the application. This leads to the question how inter-feature dependencies can
+be handled. For instance there is the lao feature in `src/features/lao` that
+manages the storage of LAO related data such as what the active LAO is. (At the
+time of writing, the application can at a specific point in time only be
+connected to one LAO). If another feature, for example the `evoting` feature,
+needs to know the ID of the active LAO in order to send a message in the correct
+channel, then it is **not** supposed to just import the corresponding function
+directly from the `lao` feature.
```

Comment:
if there are no more inter-features dependencies, don't forget to switch depcruise config to treat imports across features as errors.

Reply:
👍